### PR TITLE
Start Contiguous Architecture(WIP)

### PR DESCRIPTION
**Pull request by abhon**
I started working on the contiguous architecture and it is mostly done, but I would like to run a few tests on it before merge. @PhilipEkfeldt you might notice this when you try to use the architecture, but I missed a few of the optional parameters when making the ResidualBlocks in the network, and I have changed that so you can actually turn on batch norm and change the activation function for the Residual Blocks when making the Encoding Network. Let me know if anything sticks out.

### CODE REVIEW

**Comment by PhilipEkfeldt** [comment]
This looks good, it's basically just that change + combining it into one net?

**Comment by abhon** [comment]
Basically. I'm pretty sure it works, but if you all want I can run some tests.

File: alphazero/model.py [hunk 1]
```diff
@@ -173,3 +179,38 @@ def forward(self, encoding):
...
+        self,
+        game,
+        block,
+        num_blocks,
+        in_channels,
+        enc_channels=256,
+        activation=nn.ReLU(inplace=False),
+        batch_on=True,
+        pass_move=False,
+    ):
```
Comment (charlesoblack) [inline alphazero/model.py:199]: Can we get a docstring here describing the params?

File: alphazero/model.py [hunk 2]
```diff
@@ -173,3 +179,38 @@ def forward(self, encoding):
         value = self.valuefc2(value)
         win_value = self.tanh(value)
         return win_value
+
+
+class FinalNet(nn.Module):
```
Comment (charlesoblack) [inline alphazero/model.py:184]: this should probably be named alecnet
